addons/config/settings-config.php

```diff
@@ -39,7 +39,7 @@ public function load() {
 	 * @since 1.3.0
 	 */
 	public function add_submenu() {
-		add_options_page( 'Cookiebot Addons', __( 'Cookiebot Addons', 'cookiebot-addons' ), 'manage_options', 'cookiebot-addons', array(
+		add_submenu_page('cookiebot', 'Prior Consent', __( 'Prior Consent', 'cookiebot' ), 'manage_options', 'cookiebot-addons', array(
```

Comment:
If I am not mistaken, this means that the Cookiebot plugin has to be installed if the repository is to be used. In some cases people use Cookiebot with GTM or directly in the template. There is a checkbox for not displaying the the banner through the Cookiebot plugin. If we proceed with this our documentation on installing Cookiebot Addons should be updated.